help/datagrok/solutions/domains/bio/bio.md

```diff
@@ -302,14 +303,53 @@ To learn more about filtering, watch this [video](https://www.youtube.com/watch?
 
 ## Manage monomer libraries
 
-The default [HELM monomer library](https://github.com/datagrok-ai/public/blob/master/packages/Bio/files/monomer-libraries/HELMCoreLibrary.json) is pre-installed with the [Bio package](https://github.com/datagrok-ai/public/tree/master/packages/Bio). You can add your own monomer libraries using the dialog accessible from **Top Menu** > **Bio** > **Manage** > **Monomer Libraries**:
+Datagrok allows you to manage monomer libraries for any Macromolecule type (DNA, RNA, Peptide or custom). Monomer libraries are used to define the monomers, their properties and molecular structures that are
+used throughout the platform. They are crucial for functions like [To Atomic Level](#get-atomic-level-structure) conversion, sequence analysis, and other bioinformatics tasks, ensuring accurate representation and manipulation of macromolecules.
 
-![Monomer library file manager](./img/monomer-lib-file-manager.png "Monomer library file manager")
+The default [HELM monomer library](https://github.com/datagrok-ai/public/blob/master/packages/Bio/files/monomer-libraries/HELMCoreLibrary.json) is pre-installed with the [Bio package](https://github.com/datagrok-ai/public/tree/master/packages/Bio). You can add your own monomer libraries using the view accessible from **Top Menu** > **Bio** > **Manage** > **Monomer Libraries**:
 
-To include monomers from a library, click on a checkbox next to its name.
+![Monomer library file manager](./img/monomer-library-manager.png "Monomer library file manager")
+
+To include monomers from a library, click on a checkbox next to its name. Similarly, to exclude monomer library, uncheck the checkbox. The changes are applied immediately and reflected throughout the platform.
```

Comment:
Not exactly clear to the reader what it means to "include monomers from a library"

Reply:
what do you think of: 
To make the monomers from a library available throughout Datagrok, select the checkbox next to the library name.